.github/workflows/run-cypress-test.yml

```diff
@@ -0,0 +1,50 @@
+name: Cypress Test
+on:
+
+  # Run on any commit to the #master branch
+  push:
+    branches: [master]
+
+  # Run on pull requests into the #master branch
+
+  pull_request:
+    branches: [master]
+
+  # Allows user to trigger the workflow from GitHub's web UI
+  workflow_dispatch:
+
+jobs:
+  cypress-test:
+    runs-on: ubuntu-latest
+    steps:
+
+      # Setup
+      - name: Install AppBuilder
+        run: docker swarm init && npx digi-serve/ab-cli install ab --stack=ab --port=8080 --db.expose=false --db.password=root --db.encryption=false --tag=develop --nginx.enable=true --ssl.none --bot.enable=false --smtp.enable=false --tenant.username=admin --tenant.password=admin --tenant.email=neo@thematrix.com && cd ./ab && npm install har-validator && npm install && cd .. && mkdir ./ab/test/e2e/cypress/integration/cars_app && sudo systemctl restart docker
```

Comment:
I think you can take out har-validator now, it should be fixed in cypress. Also maybe separate the some of these commands into another step, so it's easier to understand in the future.

Reply:
Why do you need to restart docker?

Reply:
Also `mkdir` seems unnecessary. I think actions/checkout should create the directory.